library/src/main/java/ss_n/common_ktx/Permissions.kt

```diff
@@ -1,8 +1,41 @@
 package ss_n.common_ktx
 
 import android.Manifest
+import androidx.annotation.RequiresApi
 
 enum class Permissions(val permissions: Array<String>) {
+    /**
+     * [Manifest.permission.READ_MEDIA_IMAGES]
+     */
+    @RequiresApi(33)
+    IMAGE_READ(arrayOf(Manifest.permission.READ_MEDIA_IMAGES)),
+
+    /**
+     * [Manifest.permission.READ_MEDIA_VIDEO]
+     */
+    @RequiresApi(33)
+    MOVIE_READ(arrayOf(Manifest.permission.READ_MEDIA_VIDEO)),
+
+    /**
+     * [Manifest.permission.READ_MEDIA_AUDIO]
+     */
+    @RequiresApi(33)
+    AUDIO_READ(arrayOf(Manifest.permission.READ_MEDIA_AUDIO)),
+
+    /**
+     * [Manifest.permission.READ_MEDIA_IMAGES]
+     * [Manifest.permission.READ_MEDIA_VIDEO]
+     * [Manifest.permission.READ_MEDIA_AUDIO]
+     */
+    @RequiresApi(33)
+    READ_MEDIA(
+        arrayOf(
+            Manifest.permission.READ_MEDIA_IMAGES,
+            Manifest.permission.READ_MEDIA_VIDEO,
+            Manifest.permission.READ_MEDIA_AUDIO,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sasa-nori/common-ktx/blob/07511591e3c262f573f4110d358a44b5c2c346e6/library/src/main/java/ss_n/common_ktx/Permissions.kt#L35"></span>Not a valid Kotlin file (expecting an argument) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>